components/plurals/data.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
> Have we avoided it? Each ICU4X version is tied to a CLDR release

icu_datagen and icu_testdata are the only crates tied to a CLDR version.

We discussed some ideas for how to scheme the CLDR version in crates.io versions here: https://github.com/unicode-org/icu4x/issues/2540

> The globaldata feature is for clients who don't care about where the data comes from and just want ICU4X to work.

Acknowledged, but whether to use a single crate or separate crates for data is an (important) implementation detail visible  only in the lockfile. It shouldn't impact clients who don't care.

> Also keep in mind that we're releasing more frequently than CLDR, so we won't have to do data-only releases.

I hope that this won't forever be the case; once we are fully feature-complete, I want to slow down releases.

> If we put the data in separate crates, we have to stabilize the databake interface, which takes a lot of flexibility away from us.

We need to stabilize just the macros, right? We already export stable macros from databake. We're just adding some more.

Another reason to favor separate crates is that vendoring clients won't need to carry in all the big, unsafe databake files. They can disable the feature and avoid vendoring the data crates.

Do I think we need to answer this question before landing this PR? I think so, because this is a fairly fundamental decision that seems like it would be hard to decouple with future changes landing. I don't want to ship this to clients until we agree on the structure.

Reply:
> Do I think we need to answer this question before landing this PR? I think so, because this is a fairly fundamental decision that seems like it would be hard to decouple with future changes landing. I don't want to ship this to clients until we agree on the structure.

This issue is blocking 1.3 so merging does not risk shipping like this. I don't want to block the PR on this discussion, the bulk of changes is to the component APIs, whether we do `include!("../data/macros.rs")` or `use icu_globaldata::*` is not the main point.